crates/ty_python_semantic/resources/mdtest/named_tuple.md

```diff
@@ -151,9 +159,42 @@ from typing import NamedTuple
 class User(NamedTuple):
     id: int
     name: str
+    age: int | None
+    nickname: str
 
 class SuperUser(User):
-    id: int  # this should be an error
+    # this should be an error because it implies that the `id` attribute on
+    # `SuperUser` is mutable, but the read-only `id` property from the superclass
+    # has not been overridden in the class body
```

Comment:
So should there be a TODO here?

Reply:
yeah, though it's a general TODO because declaring an attribute as mutable on a subclass when there's a property on the base class, without actually overriding the property from the base class in the class body of the subclass, should be just as much an error for a non-`NamedTuple` class as it should be for a `NamedTuple` class